routes/project/location/kubernetes_cluster.rb

```diff
@@ -27,6 +27,17 @@ class Clover
           kc.incr_destroy
           204
         end
+
+        r.on "kubeconfig" do
+          r.get do
+            authorize("KubernetesCluster:create", kc.id)
+
+            content = kc.kubeconfig
+            response["Content-Type"] = "text/plain"
+            response["Content-Disposition"] = "attachment; filename=\"#{kc.name}-kubeconfig.yaml\""
+            response.write(content)
+          end
```

Comment:
huh, seems neat, but why? also I sort of loathe content-disposition attachment for something that is plain text, I kinda want to see it first pretty often rather than having it go straight to cluttering my file system when I click. But maybe I'm nuts.

Reply:
Normally, other cloud providers do that with CLI, and users don't see the kubeconfig at all. It's meant to be placed in it's default place in user home directory (by the CLI) and the user would use kubectl normally. Since we don't have those means yet, we will ask users to download and place it/or specify in their kubectl commands.

Reply:
As discussed, left the download flow as is, but moved to inline with the rest of the cluster details

![image](https://github.com/user-attachments/assets/7bf48255-6548-4332-a443-f44087769a5f)